### PR TITLE
add parent issue

### PR DESCRIPTION
**Pull request by DavidNic11**

### CODE REVIEW

File: public/jira/normalized/normalize.ts [hunk 1]
```diff
@@ -2,8 +2,27 @@ import { parseDateIntoLocalTimezone } from "../../date-helpers.js";
...
+export interface JiraIssue extends BaseIssue {
+  fields: IssueFields | LegacyFields;
+}
+
+export interface ParentIssue extends BaseIssue {
+  fields: ParentFields;
+}
+
+export interface ParentFields {
+  issuetype: { name: string; hierarchyLevel: number };
```
Comment (shoumaw) [inline public/jira/normalized/normalize.ts:19]: Should this be camel case `issueType`?

File: public/jira/normalized/normalize.ts [hunk 2]
```diff
@@ -111,7 +123,25 @@ type DefaultsToConfig<T> = {
...
+  {
+    getSummary = defaults.getSummaryDefault,
+    getHierarchyLevel = defaults.getHierarchyLevelDefault,
+    getType = defaults.getTypeDefault,
+  }: Partial<NormalizeParentConfig> = {}
+) {
+  return {
+    summary: getSummary(issue),
+    hierarchyLevel: getHierarchyLevel(issue),
+    type: getType(issue),
```
Comment (shoumaw) [inline public/jira/normalized/normalize.ts:142]: Is the status not needed as part of the normalized parent?

File: public/jira/normalized/normalize.ts (hunk 1 above)
Comment (shoumaw) [reply public/jira/normalized/normalize.ts:19]: nvm that is actually how its defined in the data

File: public/jira/normalized/normalize.ts (hunk 2 above)
Comment (DavidNic11) [reply public/jira/normalized/normalize.ts:142]: Not currently. This function is based purely off the case Justin added a while back

**Review by shoumaw — APPROVED** [review]

**Pull request merged**